### ISSUE TITLE
Investigate macro exponential explosion with multiple .modify invocations

### ISSUE THREAD

**Issue by adamw** [softwaremill/tapir#455]
See: https://github.com/ghik/silencer/issues/45

**Comment by seglo** [comment]
We have several Schema modifications while deriving the schemas. We primarily use annotations, but we've observed gaps with its support and use derived schema modifications to fill them in.  Our scala compiler build times are 100+ seconds so we'll need to investigate a different means of defining schema metadata for all our requirements.

**Comment by adamw** [comment]
@seglo thanks for the data point. If there are gaps in the annotations, can you create an issue to fill those gaps? :)

**Comment by seglo** [comment]
Of course. I apologize for the vague details I provided earlier, it's been one of those weeks. 

I created an issue that describes the issue we're seeing with the annotation API.
